docs/guide/usage.md

```diff
@@ -67,6 +67,9 @@ webview.start(custom_logic, window)
 pass
 ```
 
+If you wish to keep main thread free, you can run pywebview in a separate process. Refer to [this example](/examples/no_block.md) for details. Note that in a non-blocking mode main thread must be blocked in one way or another. Otherwise the program will quit immediately after reaching the end of main function.
```

Comment:
wrong.
> program will quit immediately after reaching the end of main function

process is not demon.
make it daemon to valid this line
```python
...Process(..., daemon=True)
```

but its a bad option because we can stop process using `Process.close()`